FirelandsProject/cataclysm
# Replacing script to generate SQL

### Changes proposed

- The script to assemble the SQL files is replaced and the corresponding folder is added to the .gitignore, so that it is not taken into account, since it is only going to assemble the files.

### Issues addressed


- Closes

### SOURCE:



### Tests performed


- Windows 10

### How to Test the Changes:


1. Download the changes, and generate the files with the script.
2. Currently, you will notice a message, that there are no updates in auth and characters, so it shows a message, but we will fix that later, adding some validation. At the moment, it is a small "bug", but it does not affect the creation of the files.
3. Test the files, in some database, to see if they run normally.

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
